panel/src/components/Forms/Blocks/BlockOptions.vue

```diff
@@ -60,6 +66,13 @@
           {{ $t("field.blocks.changeType") }}
         </k-dropdown-item>
         <hr>
+        <k-dropdown-item icon="template" @click="$emit('copy')">
+          {{ $t("copy") }}
+        </k-dropdown-item>
+        <k-dropdown-item icon="download" @click="$emit('paste')">
+          {{ $t("paste") }}
```

Comment:
I think this should be better "Paste after" to communicate where the element would be inserted